lib/sinon/stub-descriptor.js

```diff
@@ -1,41 +0,0 @@
-"use strict";
-
-var deprecated = require("./util/core/deprecated");
-var spy = require("./spy");
-var wrapMethod = require("./util/core/wrap-method");
-
-// This is deprecated and will be removed in a future version of sinon.
-// We will only consider pull requests that fix serious bugs in the implementation
-function stubDescriptor(object, property, descriptor) {
```

Comment:
This was an internal only file.

That `sinon.stub` only accepts two arguments has been [documented in this very commit](https://github.com/mroderick/sinon/blob/fdea27c9b77ea30e97bd0233853cabb7a77e0b7c/docs/release-source/release/stubs.md#var-stub--sinonstubobject-method-func)